proto/fairyring/common/shared_types.proto

```diff
@@ -32,6 +32,18 @@ message GetAggrKeyshare {
 
 message GetAggrKeyshareResponse {}
 
+// GetPrivateKeyshare defines a struct for the data payload
+message GetPrivateKeyshare {
+  string request_id = 1;
+  string identity = 2;
+  string requester = 3;
+  string rsa_64_pubkey = 4;
```

Comment:
let's make this the `secp256k1` public key instead (compressed form which is 33 bytes) so that it is compatible with cosmos addresses.

Reply:
added to fairyring. will update in fairyring client